examples/files/basic_golang_app/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/nexB/go-inspector/basicGoApp
```

Comment:
Why not make the app the main thing here? Why make it only a sample?

Reply:
@pombredanne,because they are two different applications. This is for unit testing only.